modules/jaeger-rn-new-features.adoc

```diff
@@ -4,7 +4,7 @@ Module included in the following assemblies:
 ////
 
 [id="jaeger-rn-new-features_{context}"]
-== New features {ProductName} 1.16
+= New features {ProductName} 1.16
```

Comment:
The release version is 1.17.1

Reply:
I forgot to update the attributes file on this PR (it's correct now).